### ISSUE TITLE
526ez | Implement and test 526 backup submissions

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#50111]
## Value Statement
**_As a_** veteran with an issue that prevents auto establishment of my 526 claim
**_I want to_** be sure that my submission gets to the VA, even if it is not auto-established
**_So that_** it does not go into a blackhole without my knowledge

---
## Background Context
- https://app.zenhub.com/workspaces/benefits-team-1-6138d7b57a2631001a4b7562/issues/department-of-veterans-affairs/vets-website/22367

## Acceptance Criteria
- [x] Upon final failure, pdf documents are generated
- [x] Generated Docs are sent to central mail via LH Claims intake API
- [x] Central Mail receives the items an can action the claim in some way

## Designs and Build Notes
- I have code to do all of this EXCEPT the 526 form generation itself
- EVSS thinks they can add an endpoint that returns a 526 pdf, from the same data we already submit
- Need to figure out/ensure that gets prioritized and see how long that will take vs doing our own

## Out of scope
- Add here

## Open questions
- Can EVSS add "return 526 pdf" endpoint, since they already have code that does this anyway? -they can
- How long will it take?- they did it, it didnt take that long
- Should we just do one ourselves if it will take a while?- no

## Definition of Ready 

- [x] Clear value description
- [x] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [x] Meets acceptance criteria
- [x] Passed integration testing (if applicable)
- [x] Code reviewed (internal)
- [x] Submitted to staging
- [x] Tested in staging
- [x] Released in prod
- [x] Tested via manual submission on real claims in production
- [ ] Verification of validity of claims received from VA
- [ ] Flipper turned on in production
- [ ] Observed for a few days to check logs for any unforeseen errors

**Comment by saderagsdale** [comment]
Waiting to hear from Michael Harlow/EVSS regarding when there is an endpoint available to return a PDF.

**Comment by kylesoskin** [comment]
Waiting on EVSS to get us the 526 endpoint. The rest is done (will write additional test cases in mean time)

**Comment by saderagsdale** [comment]
EVSS has the staging version of the endpoint ready. Kyle working on the test cases. @mattpointzxer0

**Comment by saderagsdale** [comment]
Waiting for EVSS to share the level of effort to move contentions out of the "overflow" area. 

Kyle discovered one potential issue with evidence uploading in additional testing. It seems that non-pdf type evidence(s) are not properly getting attached. Looking into that now before doing the additional 4 prod submissions that we will observe/track.

**Comment by saderagsdale** [comment]
Waiting for EVSS to make date timestamp changes (estimated on 1/25)

**Comment by saderagsdale** [comment]
Merged yesterday, just waiting for EVSS.

**Comment by saderagsdale** [comment]
@kylesoskin Reminder to include the other test cases that Matt raised in our last convo with him.

**Comment by kylesoskin** [comment]
EVSS staging datastamp fix released. Tested in staging, seems to work as intended.

**Comment by kylesoskin** [comment]
After discussion with Matt Self previously, we are cranking the retry window down from 15 attempt (~36hrs) to 13 attempts (~24hrs), PR above

**Comment by saderagsdale** [comment]
Future (backup submission path):

- EVSS releases today (but not until 9pm unfortunately), testing in production of 5  claims (manually, by [va.gov](http://va.gov/)) will happen tomorrow morning, I will provide submissions IDs for VA to verify
- Once confirmed as good submissions we can then either test more manually, for them to also verify, or turn it on to auto-send, or whatever they want
- [va.gov](http://va.gov/) will keep an eye on logs and dashboards for errors after it is decided to be "turned on"

**Comment by kylesoskin** [comment]
This is released in production, and turned on in the settings.yml. It is awaiting final approval before "going live" via the flipper switch.
